src/components/dashboard/dashboard.service.ts

```diff
@@ -116,7 +116,9 @@ export class DashboardService implements OnDestroy {
                 col: widget.getColumn(),
```

Comment:
Not particularly sure what the cause of this issue is, but you can see here the widget on the right is not behaving properly:

[![Image from Gyazo](https://i.gyazo.com/7ad3ae45ae36fdfe53541997a7ff20e3.gif)](https://gyazo.com/7ad3ae45ae36fdfe53541997a7ff20e3)

https://plnkr.co/edit/H4fXOruNQFsPYAHm

Reply:
I've created a separate ticket to look into this since it's a pre-existing issue